src/Step/Build/DeployStaticContent/Generate.php

```diff
@@ -93,8 +94,10 @@ public function execute(): void
             foreach ($commands as $command) {
                 $this->shell->execute($command);
             }
-        } catch (ConfigException|ShellException $exception) {
-            throw new StepException($exception->getMessage(), $exception->getCode(), $exception);
+        } catch (ShellException $e) {
+            throw new StepException($e->getMessage(), Error::BUILD_SCD_FAILED, $e);
+        } catch (ConfigException $e) {
+            throw new StepException($e->getMessage(), $e->getCode(), $e);
```

Comment:
in other steps you defined code for this exception like Error::BUILD_CONFIG_NOT_DEFINED. Why behavior is different from step to step?

Reply:
ConfigExeption has already thrown with correct ErrorCode, I'll update steps were I added it.

Reply:
I added new errors to config classes, please review. Now Config classes can throw `ConfigException` with 4 different error codes.